internal/build/pom.xml

```diff
@@ -1173,7 +1174,55 @@
                 </plugins>
             </build>
         </profile>
-
+        <profile>
+            <id>native</id>
+            <activation>
```

Comment:
all of the modules where this needs to be activated are packaging jars. can't we just use

```
<property>
        <name>packaging</name>
        <value>!pom</value>
</property>
```

and avoid more dotfiles?

Reply:
this actually does not work because maven only supports a single property for activation. sigh. maven is terrible.